sagemaker-python-sdk/mxnet_mnist/mxnet_mnist_elastic_inference.ipynb

```diff
@@ -56,7 +56,7 @@
    "source": [
```

Comment:
<p>use for loop can make this cell shorter </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2034/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>